src/main.py

```diff
@@ -161,8 +160,14 @@ def accessibility_statement(lang):
     return render_template('%s/2019/accessibility_statement.html' % (lang))
 
 
-@app.route('/sitemap.xml')
+# Handle trailing slashes for accessibility statement (as it's a special case)
+@app.route('/<lang>/accessibility-statement/')
```

Comment:
I think you should be able to decorate the existing `accessibility_statement` function with this `@app.route` and decide wither to redirect there based on trailing slash.

Reply:
@rviscomi I cannot get this to work.

I can get it to work the other way around. So if the route is setup as `/<lang>/accessibility-statement/` then flask automatically redirects any request to `/en/accessibility-statement` (without the last slash) to include the slash (with a 308 redirect). It also serves `/en/accessibility-statement/` just fine without any redirect.

However our URLs have no trailing slash (except for home pages like /en/2091/) so it's `/en/accessibility-statement` or `/en/2019/methodoloy`. And the only two ways I can get this to redirect requests to `/en/accessibility-statement/` to `/en/accessibility-statement` is either 1)  the have two routes like I have currently. Or 2) have validate catch this and redirect.

This is also explained in the Flask documentation: https://flask.palletsprojects.com/en/1.1.x/quickstart/#unique-urls-redirection-behavior

As we've been launched for a while now, and linked from many places, I don't think it's worth changing our URLs now (even with a 301 redirect), so we're left with handling this accessibility-statement edge case in the ways described above.

@rviscomi to you want to give it a go and see if I'm missing something?

Reply:
OK finally resolved this @rviscomi 
Can you check if happy now?